spec/models/application_record_spec.rb

```diff
@@ -104,6 +104,21 @@
     end
   end
 
+  describe 'interview_count' do
+    before :each do
+      record_1 = create :application_record
+      record_2 = create :application_record
+      create :interview, application_record: record_1
+      @collection = ApplicationRecord.where(id: record_1.id)
```

Comment:
(put this here because Github called my other post on an outdated diff)

Is there a reason to not just make record_1 an instance variance instead? This seems like an unnecessary database query.

Even if it was to be kept, wouldn't it be better to replace it with .find(record_1.id) or add .take to the end to account for the situation when the query returns more than one result? (abstraction for future addition to test suite since .where returns an ActiveRecord relation)